scraper/src/youtube2zim/scraper.py

```diff
@@ -1251,22 +1271,33 @@ def get_playlist_slug(playlist) -> str:
 
         # write channel.json file
         channel_data = get_channel_json(self.main_channel_id)
+        channel_data_dict = {
```

Comment:
Why not just setting the properties? Looks like you worried setting them to `None`? Not a big deal to have few extra props in JSON, and would greatly enhance readability.